rke/k8s_service_options.go

```diff
@@ -139,6 +152,13 @@ func getKubeletOptions115() map[string]string {
 	return kubeletOptions
 }
 
+func getKubeletOptions116() map[string]string {
+	kubeletOptions := getKubeletOptions()
+	kubeletOptions["authorization-mode"] = "Webhook"
+	delete(kubeletOptions, "allow-privileged")
```

Comment:
As map is being passed by ref, can you make sure by modifying it, you are not altering it globally?

Reply:
Yeah it shouldn't alter the global map, it was tested before for 1.15